frontend/javascripts/libs/render_independently.tsx

```diff
@@ -0,0 +1,42 @@
+import React from "react";
+import ReactDOM from "react-dom";
+import { document } from "libs/window";
+import { Provider } from "react-redux";
+
+type DestroyFunction = () => void; // The returned promise gets resolved once the element is destroyed.
+
+export default function renderIndependently(
+  getComponent: (arg0: DestroyFunction) => React.ReactElement<React.ComponentProps<any>, any>,
+): Promise<void> {
+  return new Promise((resolve) => {
+    import("oxalis/throttled_store").then((_Store) => {
```

Comment:
Would a normal import cause circular dependencies or why did you choose to perform the dynamic import?